tests/test_account_activation.py

```diff
@@ -39,9 +39,6 @@ def test_activate_account_invalid_token_specified(self, mock_object):
         response = self.app.get('/register/activate-account/' + token, headers=self.headers)
 
         self.assertEqual(response.status_code, 302)
```

Comment:
We should be testing more than just the status code response, We know it's redirecting yeah but we should know it's going to the right place.

If you want to go to the actual final route rather than stopping at the redirect you can add `follow_redirects=True` to the get request